src/commands/curl.js

```diff
@@ -4,6 +4,8 @@ const { parseCurlCommand } = require('curlconverter/util.js');
 const { spawn } = require('child_process');
 const { loadOAuthConf, conf } = require('../models/configuration.js');
 const { addOauthHeader } = require('@clevercloud/client/cjs/oauth.js');
+const { println } = require('../logger.js');
+const apiUrl = 'https://api.clever-cloud.com';
```

Comment:
can't we resolve it like other commands ?
https://github.com/CleverCloud/clever-tools/blob/3256552d3582f7163405e544cd3eaf57f6036982/src/models/configuration.js#L91